modules/creating-oci-infra-resources-services.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-agent-based-installer.adoc [Using the Agent-based Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-oci-infra-resources-services_{context}"]
+= Creating OCI infrastructure resources and services
+
+Before you install {product-title} on {oci-first}, you must create an {oci} environment on your virtual machine (VM) shape or bare-metal shape. By creating this environment, you can install {product-title} and deploy a cluster on infrastructure that supports a wide range of cloud options and strong security policies.
```

Comment:
Hi @pawanpinjarkar . VM shape should be removed as the Agent-based Installer route focusing on bare metal?

Reply:
I believe so but please confirm with Ramon @racedo

Reply:
No, we will install both VMs or bare metals with Agent.

Reply:
Thanks for confirming.